### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by withbest**
## Type of change

- Documentation

## Changes

-  fix some comments

## Notes

- Note 1

## Related Issues

Closes #\<issue number\>

## Checklist

- [ ] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
   - [ ] If my change requires substantial documentation changes, I have [requested support from the DevRel team](https://github.com/FuelLabs/devrel-requests/issues/new/choose)
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [ ] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [ ] I have requested a review from the relevant team or maintainers.
- [ ] I have updated the changelog to reflect the changes on this PR.

### CODE REVIEW

**Review by bitzoic — APPROVED** [review]
Thanks for your contribution!

File: docs/contributing-book/src/code_docs/naming-components.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ Using a single character to name a variable conveys little to no information to
 
 - Is this a throw away variable?
 - What is the variable meant to represent where ever it is used?
-- Does it make sense to call it by the chosen character e.g. `x` when referring to forumlas?
+- Does it make sense to call it by the chosen character e.g. `x` when referring to formulas?
 
-### Ambigious Abbreviations
```
Comment (bitzoic) [inline docs/contributing-book/src/code_docs/naming-components.md:31]: This was correct

**Review by bitzoic — CHANGES_REQUESTED** [review]

File: docs/contributing-book/src/code_docs/naming-components.md (hunk 1 above)
Comment (SwayStar123) [reply docs/contributing-book/src/code_docs/naming-components.md:31]: No, it should be Ambiguous

**Review by K1-R1 — APPROVED** [review]
Thanks!

**Pull request merged**